src/API/Google/Middleware.php

```diff
@@ -498,7 +498,7 @@ protected function new_account_name(): string {
 	 */
 	protected function default_account_name(): string {
 		return sprintf(
-			/* translators: 1: current date in the format Y-m-d */
+		/* translators: 1: current date in the format Y-m-d */
```

Comment:
Any specific reason this indent was changed? The comment belongs to the line below, so I think it should match it's indent.